packages/sentry/index.tsx

```diff
@@ -0,0 +1,17 @@
+import * as Sentry from '@sentry/nextjs';
+
+export const sentryFetch: typeof fetch = async (input, init) => {
```

Comment:
Minor point but the fetch params are officially called "resource" and "options". https://developer.mozilla.org/en-US/docs/Web/API/fetch Though options has a type of "RequestInit" according to TypeScript so who knows.

Reply:
Oh weird, typescript says they're different than mdn does
https://github.com/microsoft/TypeScript/blob/d6d26430c7a64d3a740a7705db6ad63a8bd1c59c/lib/lib.dom.d.ts#L18664

Reply:
https://github.com/iFixit/react-commerce/pull/581/commits/7ff5c5564c74db2fbfd0c1b0114f097f45861b64